charts/examples/citrix-k8s-cpx-ingress-controller/templates/citrix-k8s-cpx-ingress.yaml

```diff
@@ -4,17 +4,20 @@ metadata:
   name: cpx-ingress-k8-role
 rules:
   - apiGroups: [""]
-    resources: ["services", "endpoints", "ingresses", "pods", "secrets"]
+    resources: ["services", "endpoints", "ingresses", "pods", "secrets", "routes", "routes/status"]
```

Comment:
Wished to check if we need routes/status here? Is it intentional change?

Reply:
I think we should include  NS_NETPROFILE with yaml. What is your sugesstions @asparshk @krdpk17

Reply:
@janraj yes. Thanks for mentioning this. We will discuss with you for the required change